src/modules/env.ts

```diff
@@ -0,0 +1,2 @@
+import dotenv from 'dotenv'
+dotenv.config()
```

Comment:
You moved the function that was called on the root of the other file to this file

Reply:
It is necessary to import and configure dotenv as early as possible in the application. In every resource I looked people were doing that way.
Resources: https://www.npmjs.com/package/dotenv https://github.com/motdotla/dotenv/issues/89